docs/kb/developer-resources/updating-expiration-on-licenses.md

```diff
@@ -6,40 +6,42 @@ weight = "999999"
+For details on how to get the App ID and Channel ID, refer to [Finding Your API Token and App ID](/kb/developer-resources/finding-your-api-token-and-app-id). These values will not change and should be supplied as static values to your license updating script.
 
 ### 1) Get all outstanding licenses
+To start, get all the licenses that have been issued for a specific app:
 
-First you will get all the licenses that have been issued for a specific app:
-
-    curl -X GET \
-    	 -H 'Authorization: <YOUR-API-TOKEN>' \
-    	 https://api.replicated.com/vendor/v2/app/<YOUR-APP-ID>/licenses
-
-Note, the result will contains an array of objects for all the licenses associated with this app, including all the channels.
+```shell
+curl -X GET \
+    -H 'Authorization: <YOUR-API-TOKEN>' \
+    https://api.replicated.com/vendor/v2/app/<YOUR-APP-ID>/licenses
+```
 
 ### 2) Find expiring licenses
-
-In your language of choice you will iterate through the returned array from step 1 and create a new array that contains all licenses where "ChannelId" fields match your Beta Channel ID. You will then iterate through the new array and look for licenses with an "ExpireDate" that is within the next five days and pop those from this array.
+Iterate through the returned array from step 1 to create a new array that contains all licenses where "ChannelId" fields match your target channel ID. Select the licenses you want to extend from this array.
 
 ### 3) Update expiring licenses
-
-You will loop through the newly created array of expiring licenses from the previous step and update each of the licenses with a date that is 30 days from now. Its important to note that the field name to update on this endpoint is "expiration_date" and that you have to pass all of the expected fields from your license object because the fields will be overwritten with default values if they are left blank.
-
-      curl -X PUT \
-      	 -H 'Authorization: <YOUR-API-TOKEN>' \
-      	 -H 'Content-Type: application/json' \
-      	 https://api.replicated.com/vendor/v2/license/<YOUR-LICENSE-ID> \
-      	 -d '{"license_type":"dev",
-              "activation_email": false,
-              "airgap_download_enabled": false,
-              "assignee":"My-App",
-              "update_policy":"manual",
-              "channel_id":"<YOUR-CHANNEL-ID>",
-              "expiration_date":"2018-03-13",
-              "expiration_policy":"ignore",
-              "require_activation":false}'
+Issue a single API call to the Vendor API for each license in the array to update the expiration date. An example of this (using cURL) is:
+
```

Comment:
I think we should include the detail that they must supply all fields, or they will be set to the default value, either here or after the curl sample.